blocks/library/pullquote/editor.scss

```diff
@@ -11,6 +11,10 @@
 }
 
 .wp-block-pullquote {
+	cite {
+		display: block;
```

Comment:
This just increases the click area of the citation to the whole line (width: 100%)

Reply:
This kind of tells me that we shouldn't be using cite for this... :)